files/default/sql/ddl/updates/2.6.0.sql

```diff
@@ -134,3 +134,29 @@ SET `predictor_resources` = JSON_SET(`predictor_resources`, '$.requests', CAST(`
     `predictor_resources` = JSON_REMOVE(`predictor_resources`, '$.cores', '$.memory', '$.gpus'),
     `transformer_resources` = (CASE WHEN `transformer` IS NOT NULL then `predictor_resources` else NULL end);
 SET SQL_SAFE_UPDATES = 1;
+
+-- Add bigquery connector
+CREATE TABLE IF NOT EXISTS `feature_store_bigquery_connector`
+(
+    `id`                      int AUTO_INCREMENT,
+    `key_inode_pid` BIGINT(20) NULL,
+    `key_inode_name` VARCHAR(255) NULL,
+    `key_partition_id` BIGINT(20) NULL,
+    `parent_project`          varchar(1000) NOT NULL,
+    `dataset`                 varchar(1000) NULL,
+    `query_table`             varchar(1000) NULL,
+    `query_project`           varchar(1000) NULL,
+    `materialization_dataset` varchar(1000) NULL,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `fk_fs_storage_connector_bigq_keyfile` FOREIGN KEY (
+        `key_inode_pid`,
+        `key_inode_name`,
+        `key_partition_id`
+    ) REFERENCES `hops`.`hdfs_inodes` (`parent_id`, `name`, `partition_id`)
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+ALTER TABLE `hopsworks`.`feature_store_connector`
+    ADD COLUMN `bigquery_id` INT,
+    ADD CONSTRAINT `fs_connector_bigquery_fk` FOREIGN KEY (`bigquery_id`) REFERENCES `hopsworks`.`feature_store_bigquery_connector` (`id`) ON DELETE CASCADE;
```

Comment:
please add also `ON UPDATE NO ACTION` to be consistent with the create stable in the initial table definitions.

Reply:
added at missing places